### PR TITLE
Pass valid filenames to RuboCop and Reek

### PR DESCRIPTION
**Pull request by FooBarWidget**
Fixes bug #719.

- [x] The build passes
- [ ] TSLint is mostly happy — It was already unhappy. This PR doesn't make it _more_ unhappy.
- [ ] Prettier has been run — Not possible because TSLint was already unhappy through no fault of this PR.

### CODE REVIEW

**Review by AndreiEres — APPROVED** [review]

**Comment by FooBarWidget** [comment]
Please don't auto-close this PR. The issue is awaiting review.

**Comment by royalpeasantry** [comment]
I've recently started developing in ruby and ran into this issue.

@FooBarWidget Thank you for digging into this and submitting a fix!

This PR has been open for a few months now, unfortunately it doesn't look like any of the current approvers have write access. Does someone with write access have time to take a look?

Thanks!

**Comment by alan-pie** [comment]
@wingrunr21 or @rebornix Can you review this please to unblock #227?

**Comment by sgray** [comment]
It would really be great if someone could please review this.

**Comment by alan-pie** [comment]
Thanks @wingrunr21 !!! Any ETA on a new release?

**Comment by wingrunr21** [comment]
I have to manually QA the extension and then I will cut a release. Won't be able to do that until after work tonight.

**Comment by alan-pie** [comment]
You're a champion. Thank you again!

**Comment by benzado** [comment]
I believe this PR also fixes #578.

**Comment by robertwt7** [comment]
this is still happening now for me. `forceExclusion` doesn't seem to be a valid option anymore in `settings.json`

**Pull request merged**
